foundation.go

```diff
@@ -74,14 +92,17 @@ func InitV3Logging(appgroup, app, version, branch, revision, buildDate string) {
 	}
 
 	// set some default fields added to all logs
-	log.Logger = zerolog.New(os.Stdout).With().
+	log.Logger = zerolog.New(os.Stdout).Hook(messageIDHook{}).With().
 		Timestamp().
 		Str("logformat", "v3").
 		Str("messagetype", "estafette").
 		Str("messagetypeversion", "0.0.0").
 		Interface("source", source).
 		Logger()
 
+	// Have the error message under and object in "error" instead of in a raw string.
+	zerolog.ErrorMarshalFunc = v3ErrorMarshalFunc
```

Comment:
Can we actually inline `v3ErrorMarshalFunc`? Since it's only used in one place doesn't seem to be worth making it a variable.

Reply:
Oh, right, I didn't know that syntax :)